pkg/users/account-create-dialog.js

```diff
@@ -88,6 +92,14 @@
                     </FlexItem>
                 </Flex>
             </FormGroup>
+            <FormGroup label={_("UID")}
+                       helperTextInvalid={errors && errors.uid}
+                       validated={(errors && errors.uid) ? "error" : "default"}
+                       fieldId="accounts-create-user-uid">
+                <TextInput id="accounts-create-user-uid"
+                    onChange={value => change("uid", value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230217-135348-b75b6476-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)